src/Microsoft.Health.Fhir.Core/Features/Persistence/BadRequestException.cs

```diff
@@ -18,5 +19,16 @@ public BadRequestException(string errorMessage)
                     OperationOutcomeConstants.IssueType.Invalid,
                     errorMessage));
         }
+
+        public BadRequestException(IEnumerable<string> errorMessages)
+        {
```

Comment:
I feel that some string should be passed to the base constructor because that is what will populate the exception message. Maybe something generic like "Multiple bad request errors"?

Reply:
What about constructor parameter validation (null, non empty list, etc)? I see the BadRequestException class is public but it is used only internally so I guess it is not necessary, I'll just add a comment.

Reply:
Fixed, constructor will set exception message "Multiple bad request errors." in case of string collection containing multiple errors. If collection contains only one error, exception message will be set to the error string in the collection. No direct param validation since exception object is used only internally.

Reply:
> I'm afraid I don't understand. Message itself is not for internal use, message will be returned to the client. I meant Exception itself is used internally so there's no need for a call param validation (at least not in the RELEASE code) since we control the caller code.

Sorry for the lack of clarity! From my understanding, the message, "Multiple bad request errors.", is only for internal use. If that's correct - could we add a comment for it here?